master/internal/api_auth.go

```diff
@@ -50,7 +50,7 @@ func (a *apiServer) Login(
 		return nil, status.Error(codes.InvalidArgument, "missing argument: username")
 	}
 
-	userModel, err := user.UserByUsername(req.Username)
+	userModel, err := user.UserByUsername(context.TODO(), req.Username)
```

Comment:
Is there a reason why you're using context.TODO instead of the context that came in with Login?

Reply:
ooh good point ... I missed that